src/Microsoft.Azure.WebJobs.Host/Executors/JobHostConfigurationExtensions.cs

```diff
@@ -522,5 +523,17 @@ internal override void AddMetadata(IDictionary<string, string> metadata)
                 throw new NotSupportedException();
             }
         }
+
+        private static void ApplyConfigurationToFunctionIndex(IFunctionIndex functionIndex)
+        {
+            foreach (var functionDescriptor in functionIndex.ReadAllDescriptors())
+            {
+                var isDisabled = ConfigurationUtility.IsDisabledMethod(functionDescriptor.LogName);
+                if (isDisabled)
+                {
+                    functionDescriptor.IsDisabled = isDisabled;
+                }
```

Comment:
Can this be reduced to one line of  `functionDescriptor.IsDisabled = ConfigurationUtility.IsDisabledMethod(functionDescriptor.LogName)`?

Reply:
no, we want to set isDisabled only if it's "true". We do not want to overwrite [Disabled] attribute value.

Reply:
Oh right -- the joy of having multiple ways to change the same setting :-) So if the attribute is there, it wins?

Reply:
The app setting wins if is "true". If app setting is "false" (or another value) attribute wins.